client/packages/host/webpack.config.js

```diff
@@ -174,6 +142,13 @@ module.exports = env => {
         name: 'host',
         shared: [
           {
+            '@openmsupply-client/common': {
```

Comment:
This one had to be shared, otherwise context are not filled/shared (even though react-singleton-context is singleton and shared)